CHANGELOG

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Thu Mar  2 12:12:00 UTC 2017 - jreidinger@suse.com
+
+- allow generic set/get also on subtree (bsc#1023204)
```

Comment:
yes, it is correct. Problem in case of zypp.conf is that you need to set generic_set on subtree, so we can uncomment commented out code.